### PR TITLE
tidb: add missing labels in config file

### PR DESCRIPTION
**Pull request by glorv**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Fix https://github.com/pingcap/docs/issues/12987
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.0 (TiDB 7.0 versions)
- [x] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by qiancai** [comment]
@glorv Please assign a tech reviewer for this PR. Thanks.

**Review by nolouch — APPROVED** [review]
lgtm

File: tidb-configuration-file.md [hunk 1]
```diff
@@ -213,6 +213,16 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
 + 默认值：false
 + 表级锁用于协调多个 session 之间对同一张表的并发访问。目前已支持的锁种类包括 `READ`、`WRITE` 和 `WRITE LOCAL`。当该配置项为 `false` 时，执行 `LOCK TABLES` 和 `UNLOCK TABLES` 语句不会生效，并且会报 "LOCK/UNLOCK TABLES is not supported" 的警告。更多信息，请参考 [`LOCK TABLES` 和 `UNLOCK TABLES`](/sql-statements/sql-statement-lock-tables-and-unlock-tables.md)。
 
+### `labels`
```
Comment (qiancai) [inline tidb-configuration-file.md:216]:
```suggestion
### `labels` <span class="version-mark">从 v6.3.0 版本开始引入</span>
```

File: tidb-configuration-file.md [hunk 2]
```diff
@@ -213,6 +213,16 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
...
+
++ 指定服务器标签，例如 `{ zone = "us-west-1", dc = "dc1", rack = "rack1", host = "tidb1" }`。
++ 默认值：`{}`
+
+> **注意：**
+>
+> 1. 标签 `zone` 在 TiDB 中具有特殊用途，用于指定服务器的所在的区域信息，当设置此标签为非空值时，对应的值会被自动用于 [`txn-score`](/system-variables.md#txn_scope) 和 [`Follower read`](/follower-read.md) 等功能。
+> 2. 标签 `group` 在 TiDB-Operator 中具有特殊用途，如果使用 TiDB-Operator 部署集群，最好不要手动指定此标签。
```
Comment (qiancai) [inline tidb-configuration-file.md:224]:
```suggestion
> - 标签 `zone` 在 TiDB 中具有特殊用途，用于指定服务器所在的区域信息，当设置 `zone` 为非空值时，对应的值会被自动用于 [`txn-score`](/system-variables.md#txn_scope) 和 [`Follower read`](/follower-read.md) 等功能。
> - 标签 `group` 在 TiDB-Operator 中具有特殊用途。对于使用 TiDB-Operator 部署集群，建议不要手动指定此标签。
```

File: tidb-configuration-file.md (hunk 1 above)
Comment (glorv) [reply tidb-configuration-file.md:216]: labels 很早就支持了，6.3 是在 tidb 里面加了 http 的接口支持动态修改

**Review by qiancai — APPROVED** [review]
LGTM

File: tidb-configuration-file.md [hunk 3]
```diff
@@ -213,6 +213,16 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
...
 
+### `labels`
+
++ 指定服务器标签，例如 `{ zone = "us-west-1", dc = "dc1", rack = "rack1", host = "tidb1" }`。
++ 默认值：`{}`
+
+> **注意：**
+>
+> - 标签 `zone` 在 TiDB 中具有特殊用途，用于指定服务器所在的区域信息，当设置 `zone` 为非空值时，对应的值会被自动用于 [`txn-score`](/system-variables.md#txn_scope) 和 [`Follower read`](/follower-read.md) 等功能。
+> - 标签 `group` 在 TiDB Operator 中具有特殊用途。对于使用 TiDB Operator 部署集群，建议不要手动指定此标签。
```
Comment (ran-huang) [inline tidb-configuration-file.md:224]:
```suggestion
> - 标签 `group` 在 TiDB Operator 中具有特殊用途。对于使用 [TiDB Operator](/tidb-operator-overview.md) 部署集群，建议不要手动指定此标签。
```

File: tidb-configuration-file.md [hunk 4]
```diff
@@ -213,6 +213,16 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
...
 
+### `labels`
+
++ 指定服务器标签，例如 `{ zone = "us-west-1", dc = "dc1", rack = "rack1", host = "tidb1" }`。
++ 默认值：`{}`
+
+> **注意：**
+>
+> - 标签 `zone` 在 TiDB 中具有特殊用途，用于指定服务器所在的区域信息，当设置 `zone` 为非空值时，对应的值会被自动用于 [`txn-score`](/system-variables.md#txn_scope) 和 [`Follower read`](/follower-read.md) 等功能。
+> - 标签 `group` 在 TiDB Operator 中具有特殊用途。对于使用 [TiDB Operator](/tidb-operator-overview.md) 部署集群，建议不要手动指定此标签。
```
Comment (qiancai) [inline tidb-configuration-file.md:224]:
```suggestion
> - 标签 `group` 在 TiDB Operator 中具有特殊用途。对于使用 [TiDB Operator](/tidb-operator-overview.md) 部署的集群，建议不要手动指定此标签。
```

**Pull request merged**